desc/objectives/_geometry.py

```diff
@@ -1077,3 +1078,224 @@ def compute(self, *args, **kwargs):
             profiles=constants["profiles"],
         )
         return data["L_grad(B)"]
+
+
+class PlasmaCoilFeasibilty(_Objective):
```

Comment:
feel like this name is too general, since there are a lot of different coil feasibility metrics (like the LgradB above). Might want something more specific to this.

Reply:
Yea I could not think of a good one. maybe one like BStrengthDistanceFeasibility or something...

Reply:
What about `BDistanceVariance`?